va/va.go

```diff
@@ -285,6 +286,14 @@ func NewValidationAuthorityImpl(
 		return nil, errors.New("no account URI prefixes configured")
 	}
 
+	for i, va1 := range remoteVAs {
+		for j, va2 := range remoteVAs {
+			if i != j && va1.Perspective == va2.Perspective && va1.Perspective != "" {
+				return nil, fmt.Errorf("duplicate remote VA perspective %q", va1.Perspective)
+			}
+		}
+	}
```

Comment:
I think a check like this could give someone a false sense of confidence, when in reality the backends it starts with will be steadily replaced as remote VAs scale up and down daily. It probably doesn't need to go away, but we should comment that these are also evaluated when receiving each validation/check RPC.